images/spin-multi-trigger-app/spin-redis-trigger/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM --platform=${BUILDPLATFORM} rust:1.72 AS build
+FROM --platform=${BUILDPLATFORM} rust:1.85 AS build
 WORKDIR /opt/build
 COPY . .
-RUN rustup target add wasm32-wasi && cargo build --target wasm32-wasi --release
+RUN rustup target add wasm32-wasip1 && cargo build --target wasm32-wasip1 --release
```

Comment:
Same as above. In this case you would also need to split this line in two different RUN commands.